tests/data/sample_specification.yaml

```diff
@@ -16,22 +16,20 @@ speech_acts:
     content_bool: pt:bool
     content_str: pt:str
   performative_pct:
-#    content_set_ct: pt:set[ct:DataModel] # custom type inside of set, list, and dict isn't allowed.
```

Comment:
Either keep this comment or have a `content_set_ct: pt:set[ct:DataModel]` content if it is now allowed.

Reply:
Also use a different name to DataModel.

Reply:
Still missing this.

Reply:
`# content_set_ct: pt:set[ct:DataModel] # custom type inside of set isn't allowed.`
in line 24. sets does not support. so this comment stays